spanner/transaction.go

```diff
@@ -357,16 +369,20 @@ type QueryOptions struct {
 
 	// The request tag to use for this request.
 	RequestTag string
+
+	// If this is for a partitioned query and DataBoostEnabled field is set to true, the request will be executed via Spanner independent compute resources.
```

Comment:
Same as above: split over two lines and add an extra sentence that it has no effect on regular reads.

Reply:
Added